lib/redisFactory.js

```diff
@@ -21,9 +21,12 @@ exports.job = (opt) => {
     redis = new Redis({
       sentinels: getSentinels(opt.redisSentinelForJob),
       name: opt.redisSentinelNameForJob,
+      password: opt.redisPassword || null    // null is default in ioredis
```

Comment:
do we need `|| null`, can we keep it as `password: opt.redisPassword`?

Reply:
updated